sdk/src/main/java/com/adzerk/android/sdk/AdzerkSdk.java

```diff
@@ -393,6 +397,22 @@ private AdzerkService getAdzerkService() {
         return service;
     }
 
+    private static class SdkVersionRequestInterceptor implements Interceptor {
+
+        static String SDK_VERSION_HEADER = "X-Adzerk-Sdk-Version";
+        static String SDK_VERSION = "adzerk-android-sdk:" + BuildConfig.VERSION_NAME;
```

Comment:
Since we're going to have both Decision and Management SDKs reporting at some point in the future, it'd be great if we could announce which of those is being reported in the string. Following the format in the other decision SDKs would look something like `adzerk-decision-sdk-android:*version*`.